### DISCUSSION TITLE
GeoIPLookup enablement?

### DISCUSSION THREAD

**Discussion by XxTRAINEExX** [HomeITAdmin/nextcloud_geoblocker#55] [Q&A]
I followed the instructions for installing GEOIPLookup on the server but still get the error below:

"Geoiplookup": ERROR: Service seem to be not installed on the host of the Nextcloud server or not reachable for the web server or is wrongly configured (is the database for IPv4 and IPv6 available?!). Maybe the use of the php function exec() is disabled in the php.ini.

I verified the php.ini located in the snap package does not have exec() listed in the disabled_functions. Are there other steps to make this work? Could it be something to do with the snap package being isolated from the underlying commands in the OS?

Brian

**Comment by HomeITAdmin** [comment]
I am not so much into snap but I think with your last question you could be on the right direction. 
Can you call geoiplookup from within the package?

Another direction could be to use another service. Maybe it is easier to fulfill the preconditions for the RIR Data service.
